### DISCUSSION TITLE
Wallet crashed

### DISCUSSION THREAD

**Discussion by Thegreenpiranha** [GingerPrivacy/GingerWallet#117] [General]
while trying to download transaction history and continues to do so afer closing and reopening the wallet

error code:

Exception type: System.InvalidOperationException

Message: An attempt was made to transition a task to a final state when it had already completed.

Stack Trace:    at WalletWasabi.Fluent.Desktop.Program.Main(String[] args)

Inner Exception:

**Comment by luzius1089** [comment]
Hello there,

I want to help you but I will need more details. The error code is too general. Can you help me reproduce the issue so that I can debug it? Which version are you using? The error happens randomly or whenever you do a series of actions?

**Comment by luzius1089** [comment]
Moved here: https://github.com/GingerPrivacy/GingerWallet/issues/119
